src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/RazorLSPConventions.cs

```diff
@@ -35,7 +35,7 @@ public static Uri GetRazorDocumentUri(Uri virtualDocumentUri)
                 throw new ArgumentNullException(nameof(virtualDocumentUri));
             }
 
-            var path = virtualDocumentUri.GetAbsoluteOrUNCPath();
```

Comment:
I noticed this wasn't working well for paths that look like `file:///blah/blah.razor`. Given we just want to recreate the Uri with just the suffix changed, I think this change is fine.